itl/plugins-contrib.d/storage.conf

```diff
@@ -56,3 +56,49 @@ object CheckCommand "glusterfs" {
 	vars.glusterfs_inode_warning = 90
 	vars.glusterfs_inode_critical = 95
 }
+
+object CheckCommand "ceph" {
+	command = [ PluginDir + "/check_ceph.py" ]
+
+	arguments = {
+		"-e" = {
+			value = "$service.vars.ceph_exec_dir$"
```

Comment:
Following other CheckCommands in the ITL, the _value_ attribute should be only `ceph_exec_dirs`. This way one can use `vars.ceph_exec_dirs` as a custom variable  to adjust the attribute. 

Same for the other argument value attributes.

Reply:
Good catch, must have been a caffeine underdose on my part ^_^